src/Schema/OracleSchemaManager.php

```diff
@@ -320,19 +323,226 @@ private function getQuotedIdentifierName($identifier)
         return $identifier;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function listTables()
+    {
+        $currentDatabase = (string) $this->_conn->getDatabase();
+
+        $columnsData     = $this->groupObjectRecordsByTable(
+            $this->getDatabaseColumns($currentDatabase),
+            'TABLE_NAME'
+        );
+        $foreignKeysData = $this->groupObjectRecordsByTable(
+            $this->getDatabaseForeignKeys($currentDatabase),
+            'TABLE_NAME'
+        );
+        $indexesData     = $this->groupObjectRecordsByTable(
+            $this->getDatabaseIndexes($currentDatabase),
+            'TABLE_NAME'
+        );
+
+        $tables = [];
+
+        foreach (array_keys($columnsData) as $tableName) {
+            $columns = $this->_getPortableTableColumnList(
+                $tableName,
+                '',
```

Comment:
Why is this hard-coded to an empty string?

Reply:
changed to `$currentDatabase`